src/features/inventory/InventoryPreview.tsx

```diff
@@ -0,0 +1,90 @@
+import { useActor } from "@xstate/react";
+import classNames from "classnames";
+import React, { useContext, useState } from "react";
+
+import { Box } from "components/ui/Box";
+import { OuterPanel } from "components/ui/Panel";
+import { Context } from "features/game/GameProvider";
+import { InventoryItemName } from "features/game/types/game";
+import { ITEM_DETAILS } from "features/game/types/images";
+import { Craftable } from "features/game/types/craftables";
+import { SeedName, SEEDS } from '../game/types/crops';
+import { NFTs, TOOLS, FOODS } from "features/game/types/craftables";
+
+interface Props {
+    items: Object
```

Comment:
I would suggest avoid using object as this does not catch any type errors. A developer could change the input and accidentally break something.

Consider extracting the types you know that `items` need to have.

For instance.

```
interface Props {
    items: {
        name: InventoryItemName
        description: string
        image: any
    }[]
}
```

This also helps catches any bugs in this component with typos.